### DISCUSSION TITLE
Fragrance level or hours remaining

### DISCUSSION THREAD

**Discussion by megavolt76** [natekspencer/hacs-pura#55] [Ideas]
The main reason I found this project was to be able to monitor and alert when fragrances are low. I see there is a runtime for the fragrances but based on intensity the time left is an unknown variable. In the app it shows the oil level left based on hours. Is there any way to get this information into your project or at least a vial-empty attribute?

**Comment by natekspencer** [comment]
> I see there is a runtime for the fragrances but based on intensity the time left is an unknown variable.

There isn't anything in the data to indicate oil level or time left unfortunately. To add one would basically just be logic around when the total runtime passes a certain value. However, because runtime is available as it's own sensor, I feel like it's a bit redundant to create a separate entity for that since you can already create automations and/or alerts for when the runtime reaches a certain value.
